data/sql/updates/pending_db_world/rev_1548117070754986000.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1548117070754986000');
+
+DELETE FROM `command` WHERE `name` IN ('spy follow', 'spy unfollow', 'spy groupfollow', 'spy groupunfollow', 'spy clear', 'spy status');
+INSERT INTO `command` (`name`,`security`,`help`) VALUES
+('spy follow',          2, 'Syntax: .spy follow [$name]\nStarts reading all the private conversations of that player.'),
```

Comment:
why are there [] around "$name" ? is it optional?

example in our db: `Syntax: .account delete $account`

Reply:
Fixed

Reply:
ok :+1: